linera-execution/src/system.rs

```diff
@@ -609,7 +609,7 @@ where
                     let message = RawOutgoingMessage {
                         destination: Destination::Recipient(account.chain_id),
                         authenticated: false,
-                        is_protected: true,
+                        kind: MessageKind::Tracked,
```

Comment:
My comment about the fungible app probably also applies to native token credits. Maybe we should check whether they are bounced, and if so, credit the tokens to the authenticated signer instead of the target account owner?